README.md

```diff
@@ -60,3 +60,17 @@ in `metatx/ERC2771ContextUpgradeable.sol` from a private to a public address.
 There is another issue related to the `_trustedForwarder` in that contract that has not yet been resolved.
 
 Since `aKLIMA` was a prelaunch coupon and the existing deployed versino of the redemption contract works as expected, this contract is no longer needed.
+
+## Documentation
+
+This repo is configured with [Foundry](https://book.getfoundry.sh/) to generate documentation for the solidity source files with the [forge doc](https://book.getfoundry.sh/reference/forge/forge-doc) command.
+
+Foundry forge doc generates and builds an mdbook from the natspec comments contained within the Solidity source files.The `book.toml` file located in the root folder contains the config settings and the `preprocess_summary.py` script allows you to customize the book after it is generated.
+
+*If you have not installed Foundry, follow the guide above before proceeding.*
+
+**Please follow these rules:**
+- Annonate your code with [proper NatSpec comments](https://docs.soliditylang.org/en/latest/natspec-format.html)
+- Before committing code, run the `force doc --build` in order to build the doc/book
```

Comment:
Type to use the command `forge` not `force`

Reply:
Will get this fixed and add more from my notes too :+1: